src/components/InfoComponents/StoreList.js

```diff
@@ -119,18 +119,47 @@ const CategoryImage = styled.img`
   height: 57px;
   border-width: 50%;
   margin: 0 auto 13px;
-  
+
   @media (max-width: 576px) {
     width: 50px;
     height: 50px;
   }
 `;
-
+const SearchCategory = styled.div`
+  text-align: center;
+  position: relative;
+`;
+const Input = styled.input`
+  font-family: NanumBarunGothic0TF;
```

Comment:
요 폰트는 NanumBarunGothic0TF  가 아니라 App.js 에 보시면 font-family: 'NanumBarunGothic' 로 설정해두어서 
```suggestion
  font-family: NanumBarunGothic;
```
로 고치면 좋을것 같네요